app/controllers/alchemy/pages_controller.rb

```diff
@@ -116,6 +116,9 @@ def load_page
 
     # Redirects to given url with 301 status
     def redirect_permanently_to(url)
+      # to avoid hitting browser cache on your local development machine
+      return redirect_to(url) unless Rails.env.development?
```

Comment:
Wouldn't this be

```suggestion
      return redirect_to(url) if Rails.env.development?
```

to avoid the 301 status caching on dev machines?

Reply:
absolutely! :man_facepalming: Sorry, I never thought that I'm able to include an error by extracting a single line! :laughing: